src/CoreMidi/MidiThruConnectionParams.cs

```diff
@@ -327,19 +327,19 @@ internal NSData WriteStruct ()
 				Marshal.StructureToPtr (connectionParams, buffer, false);
 
 				if (connectionParams.NumControlTransforms > 0) {
-					unsafe { // Lets use memcpy to avoid a for loop
-						fixed (MidiControlTransform* arrAddr = &Controls[0])
-							Runtime.memcpy (bufferEnd, (IntPtr) arrAddr, controlsSize * connectionParams.NumControlTransforms);
+					unsafe {
+						fixed (void* arrAddr = &Controls[0])
+							Buffer.MemoryCopy ((void*) bufferEnd, arrAddr, controlsSize * connectionParams.NumControlTransforms, controlsSize * connectionParams.NumControlTransforms);
```

Comment:
Buffer.MemoryCopy has the source/destination parameters reversed wrt memcpy, but they're the same order here.